### DISCUSSION TITLE
Voicemail

### DISCUSSION THREAD

**Discussion by VOGUK** [mikopbx/Core#832] [General ( Обо всём )]
Hi, I've installed MIKOPBX and so far it's great and really simple to use.

I was wondering though, how to I add voicemail to the extentions? I can't seem to see it anywhere in the manual.

Is this feature available?

**Comment by boffart** [comment]
There is a voicemail "service extension" in the incoming routes. Use it

See the description on the link (use google translate)
https://wiki.mikopbx.ru/mail-settings#golosovaja_pochta

- A voicemail will be sent to each employee who missed the call (if the email is specified in his card).
- Voicemail will always be sent to the "Single Email for sending Voicemail messages"
- It is possible to listen to the voicemail recording from the PBX call history

**Reply by batboyboy1990** [reply]
if only they could listen to it again by typing a certain number, it would be better!
